src/FEValues/CellValues.jl

```diff
@@ -68,6 +68,12 @@ end
 function Base.copy(cv::CellValues)
     return CellValues(copy(cv.fun_values), copy(cv.geo_mapping), copy(cv.qr), _copy_or_nothing(cv.detJdV))
 end
+function task_local(cv::CellValues)
+    return CellValues(
+        task_local(cv.fun_values), task_local(cv.geo_mapping), task_local(cv.qr),
```

Comment:
It is an invariant that CellValues' qr does not change, so we do not need to task_local qr here right?

Reply:
> It would allow task_local(pv::PointValues) = PointValues(task_local(pv.cv)) which would be nice, and I think there wouldn't be any significant cost of doing this?

Good point.